src/services/database.ts

```diff
@@ -24,8 +22,8 @@ export const createDbConnection = () => {
 };
 
 /* Closes current connection to MongoDB */
-export const disconnectDatabase = () => {
-  const db = mongoose.connection;
+export const disconnectDatabase = (): void => {
+  const db: mongoose.Connection = mongoose.connection;
```

Comment:
nit: this doesn't need to be annotated since `mongoose.connection` already has the type `Connection`
```suggestion
  const db = mongoose.connection;
```